app/celery/tasks.py

```diff
@@ -30,3 +30,13 @@ def send_sms(service_id, notification_id, encrypted_notification):
 
     except SQLAlchemyError as e:
         current_app.logger.debug(e)
+
+
+@notify_celery.task(name='send-sms-code')
+def send_sms_code(encrypted_notification):
+    notification = encryption.decrypt(encrypted_notification)
```

Comment:
Is this a 'notification'? Should it be called something like 'validation code'. Notifications have a specific meaning in the code and we have loads of confusion, might be useful to tidy up our domain language.